public/components/freelancer-create.js

```diff
@@ -131,6 +130,30 @@ class CreationForm extends React.Component {
           </div>
 
           <div className="group">
+            <input ref="description" className="job-description" name="job-descritpion" type="text" required/>
+            <span className="bar"></span>
+            <label>
+              Job Description
+            </label>
+          </div>
+
+          <div className="group">
+            <input ref="tags" className="job-tags" name="job-tags" type="text"/>
```

Comment:
Maybe we should check that the tags are entered actually as comma separated tags. We could do it with a regex, like for the phone number.